python/strings_udf/strings_udf/lowering.py

```diff
@@ -160,18 +185,27 @@ def cast_string_view_to_udf_string(context, builder, fromty, toty, val):
     "udf_string_from_string_view",
     types.void(types.CPointer(string_view), types.CPointer(udf_string)),
 )
+# utilities
+_create_string_view_from_udf_string = cuda.declare_device(
+    "string_view_from_udf_string", types.void(_UDF_STRING_PTR, _STR_VIEW_PTR)
+)
 
 
 def call_create_udf_string_from_string_view(sv, udf_str):
     _create_udf_string_from_string_view(sv, udf_str)
 
 
+def call_create_string_view_from_udf_string(udf_str, sv):
+    _create_string_view_from_udf_string(udf_str, sv)
+
+
 # String function implementations
 def call_len_string_view(st):
     return _string_view_len(st)
 
 
 @cuda_lower(len, string_view)
+@cuda_lower(len, udf_string)
```

Comment:
Same question as before for why we're adding the len overload as part of this PR. I'm guessing that somewhere we need it to determine how many bytes to copy or something?